### PR TITLE
Reuse the LESS variable for white color across all stylesheets

### PR DESCRIPTION
**Pull request by TheCrowsJoker**
I have added variables to the plugin files to make creating your own theme easier. The variables are all linked back to the _colors.less file.

### CODE REVIEW

**Comment by mattab** [comment]
Hi @TheCrowsJoker Thanks for this great Pull request!

Ping @tzi  FYI: a new dark theme by @TheCrowsJoker  has been published on the Marketplace: http://themes.piwik.org/ - and we've removed/killed PleineLune :-)

**Comment by tzi** [comment]
Thanks @TheCrowsJoker!
This is a really nice realisation :+1: 

Cheers,
Thomas.

File: plugins/Login/Login.php [hunk 1]
```diff
@@ -41,6 +42,12 @@ public function getJsFiles(&$jsFiles)
         $jsFiles[] = "plugins/Login/javascripts/login.js";
     }
 
+   public function getStylesheetFiles(&$stylesheetFiles)
+    {
+        $stylesheetFiles[] = "plugins/Login/stylesheets/login.less";
+        $stylesheetFiles[] = "plugins/Login/stylesheets/variables.less";
```
Comment (sgiehl) [inline plugins/Login/Login.php]: Did you forget to add the file 'variables.less'. Doesn't exist in current master. I'll remove that line. Please readd it if that file was just missing.

**Comment by sgiehl** [comment]
I'll remove the include now as I need the builds running. Please readd it if required.

**Pull request merged**